### PR TITLE
Add alternative _.compact

### PR DESCRIPTION
**Pull request by inuscript**
Append compact altanative function

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ For more information, see [Configuring the ESLint Plugin](configuring.md)
 1. [_.lastIndexOf](#_lastindexof)
 1. [_.reverse](#_reverse)
 1. [_.without](#_without)
+1. [_.compact](#_compact)
```
Comment (inuscript) [inline README.md]: Should I sort list with alphanumerical?

**Comment by cht8687** [comment]
could be better that we also update the rules for this.

File: README.md (hunk 1 above)
Comment (stevemao) [inline README.md]: I think so...

**Comment by stevemao** [comment]
@cht8687 did you mean eslint rule?

**Comment by cht8687** [comment]
@stevemao  yes.

**Comment by stevemao** [comment]
I think it's complicated because you can't just swap `_.compact` with `array.filter`. All other rules can be directly swapped without changing the function.

**Comment by inuscript** [comment]
I refer this from `_.without` and it's has not eslint rule.

**Comment by stevemao** [comment]
I think we can have the eslint rule discussion in a different issue.

File: README.md (hunk 1 above)
Comment (stevemao) [inline README.md]: Can you fix the order and we'll merge this. Thanks :)

Comment (inuscript) [inline README.md]: Fix it!

Comment (cht8687) [inline README.md]: thanks.

**Pull request merged**